src/org/zaproxy/zap/extension/authenticationstatus/AuthenticationChecklistPanel.java

```diff
@@ -0,0 +1,217 @@
+package org.zaproxy.zap.extension.authenticationstatus;
+
+import java.awt.GridBagLayout;
+import java.util.regex.Pattern;
+
+import javax.swing.BorderFactory;
+import javax.swing.Icon;
+import javax.swing.ImageIcon;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.Border;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.Constant;
+import org.zaproxy.zap.authentication.AuthenticationMethod;
+import org.zaproxy.zap.authentication.ManualAuthenticationMethodType.ManualAuthenticationMethod;
+import org.zaproxy.zap.model.Context;
+import org.zaproxy.zap.users.User;
+import org.zaproxy.zap.view.LayoutHelper;
+
+public class AuthenticationChecklistPanel extends JPanel {
+
+	private static final long serialVersionUID = 8228766560419086082L;
+	
+	private static final Logger logger = Logger.getLogger(AuthenticationChecklistPanel.class);
+
+	private final JLabel contextStatusLabel;
+	private final JLabel userStatusLabel;
+	private final JLabel authenticationMethodStatusLabel;
+	private final JLabel loggedInOutIndicatorStatusLabel;
+
+	private static final Icon SUCCESSFULLY_CONFIGURED_STATUS_ICON;
+	private static final Icon CONFIGURATION_NOT_VALIDATED_YET_ICON;
+	private static final Icon BAD_CONFIGURATION_ICON;
+	private static final Icon MINIMUM_CONFIGURATION_ICON;
+
+	private boolean configuredSuccessfully = false;
+
+	static {
+		SUCCESSFULLY_CONFIGURED_STATUS_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/tick-circle.png"));
+		CONFIGURATION_NOT_VALIDATED_YET_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/question-white.png"));
+		BAD_CONFIGURATION_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/cross-circle.png"));
+		MINIMUM_CONFIGURATION_ICON = new ImageIcon(
+				AuthenticationChecklistPanel.class.getResource("/org/zaproxy/zap/extension/authenticationstatus/resources/exclamation-circle.png"));;
+	}
+
+	public AuthenticationChecklistPanel() {
+		contextStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.context"));
+		userStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.user"));
+		authenticationMethodStatusLabel = new JLabel(
+				Constant.messages.getString("authenticationstatus.dialog.verification.label.authentication"),
+				JLabel.LEFT);
```

Comment:
That's controlled by the layout, when adding the components you need at least one column with a non-zero `weightx`, otherwise the extra space is added to the edges (thus centring the labels).

Reply:
thanks, haven't worked with UI before... I moved to `GridLayout` and LGTM.. Hope I will not need `GridBagLayout` for the `AuthenticationChecklistPanel`

Reply:
OK, yeah, this case does not require a fancy layout :)